lib/pages/contact/contact.dart

```diff
@@ -34,189 +55,223 @@ class _ContactPageState extends State<ContactPage> {
           backgroundColor: kBlueGrey,
           leading: Container(),
           leadingWidth: 0,
-          title: Text(
-              'Contacts',
-              style: MyFonts.w500.size(20).setColor(kWhite)
-          ),
+          title:
+              Text('Contacts', style: MyFonts.w500.size(20).setColor(kWhite)),
           actions: [
             IconButton(
-                onPressed: () {Navigator.of(context).pop();},
+                onPressed: () {
+                  Navigator.of(context).pop();
+                },
                 icon: Icon(IconData(0xe16a, fontFamily: 'MaterialIcons')))
           ],
         ),
-        body: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Padding(
-              padding: const EdgeInsets.fromLTRB(8,14,8,14),
-              child: ContactSearchBar(),
-            ),
-            /*Padding(
-              padding: const EdgeInsets.fromLTRB(0, 0,0,10),
-              child: Row(
-                children: [
-                  Expanded(flex: 16, child: Container(),),
-                  Expanded(
-                    flex: 106,
-                    child: Container(
-                      height: 100,
-                      decoration: BoxDecoration(
-                        borderRadius: BorderRadius.circular(20),
-                        color: lGrey,
-                      ),
-                      child: Column(
-                        mainAxisAlignment: MainAxisAlignment.center,
-                        children: [
-                          Icon(Icons.warning, color: kGrey8,),
-                          Text('Emergency', style: MyFonts.medium.size(10).setColor(kWhite),),
-                        ],
-                      ),
-                    ),
-                  ),
-                  Expanded(flex: 5, child: Container(),),
-                  Expanded(
-                    flex: 106,
-                    child: Container(
-                      height: 100,
-                      decoration: BoxDecoration(
-                        borderRadius: BorderRadius.circular(20),
-                        color: lGrey,
-                      ),
-                      child: Column(
-                        mainAxisAlignment: MainAxisAlignment.center,
-                        children: [
-                          Icon(Icons.directions_bus, color: kGrey8,),
-                          Text('Transport', style: MyFonts.medium.size(10).setColor(kWhite),),
-                        ],
+        body: Provider<ContactStore>(
+          create: (context) => ContactStore(),
+          builder: (context, _) {
+            return Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(8, 14, 8, 14),
+                  child: ContactSearchBar(),
```

Comment:
@Hareesh-Nandigrama Current this searches across `ContactModel`s. I guess searching `ContactDetailsModel`s will be more useful. Or maybe combine both if that's feasible?